lesson_source/0042/pl.md

```diff
@@ -0,0 +1,160 @@
+1.22 Deployment aplikacji webowej
+=================================
+
+Serwer WWW
+----------
+
+Do tej pory, kiedy uruchamialiśmy aplikacje webowe napisane we Flasku, używaliśmy polecenia:
+
+    :::python3
+    app.run()
+
+W rzeczywistości powoduje ono uruchomienie tak zwanego **serwera WWW**. Serwer WWW to aplikacja, której zadaniem jest
+przyjmowanie i obsługa zapytań HTTP. W większości przypadków, z którymi mieliśmy do czynienia do tej pory,
+serwer przekazywał zapytania do naszej aplikacji, która zajmowała się np. pobraniem danych z bazy i wygenerowaniem
+strony HTML, będącej właściwą odpowiedzią na dane zapytanie. W rzeczywistości wiele zapytań może nie wymagać
+przekazania ich do samej aplikacji, przykładowo:
+- zapytania o pliki statyczne (obrazki, pliki CSS, JavaScript) - takie pliki są przekazywane bez żadnych zmian
+jako odpowiedź do klienta (przeglądarki), więc nie ma sensu, żeby były przetwarzane przez naszą aplikację;
+- zapytania, które wymagają długiego przetwarzania mogą być cachowane po stronie serwera WWW - jeśli takie samo
+zapytanie pojawi się kilka razy, serwer może wykorzystać odpowiedź z cache'u bez kontaktowania się z aplikacją.
+
+Serwery WWW mają też różne inne możliwości, między innymi:
+- uwierzytelnianie - możemy wymagać, aby dostęp do całej aplikacji wymagał podania nazwy użytkownika i hasła,
+dzięki temu nie musimy implementować tego mechanizmu w aplikacji (uwaga - po stronie serwera możemy zabezpieczyć
+dostęp tylko do całej aplikacji - jeśli chcemy np. mieć różne uprawnienia dla użytkowników, musimy to zaimplementować);
```

Comment:
musimy to zaimplementować sami/musimy to zaimplementować po jej stronie (przynajmniej IMHO)